packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DataVisAnnotationTask/DataVisAnnotationTask.js

```diff
@@ -0,0 +1,96 @@
+import counterpart from 'counterpart'
+import { Text } from 'grommet'
+import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
+import { observable } from 'mobx'
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Markdownz } from '@zooniverse/react-components'
+import TaskInputField from '../TaskInputField'
+import InputStatus from '../InputStatus'
+import InputIcon from '../InputIcon'
+import Graph2dRangeXIcon from './components/Graph2dRangeXIcon'
+
+import en from './locales/en'
+counterpart.registerTranslations('en', en)
+
+export const StyledFieldset = styled.fieldset`
+  border: none;
+`
+
+function storeMapper (stores) {
+  const annotations = stores.classifierStore.classifications.currentAnnotations
+  const { active, setActive } = stores.classifierStore.dataVisAnnotating
```

Comment:
Not a blocking comment but should task stores be children of the classifications store, or maybe attached to a generic tasks store rather than root? I can see our flat store structure getting hard to manage as more tasks are added and it grows outwards.

Reply:
Yeah I can see us breaking that up like I did with the WorkflowStore and WorkflowStepStore. Annotations could be its own store handling the add/remove annotation calls.

Reply:
I think I'd go for `classifierStore.tasks.dataVisAnnotating`. Similarly, I think workflow steps could be `classifierStore.workflow.steps` which would give the workflow store access to actions on the workflowStep store. At the moment, the structure's completely flat, and I'm thinking that loses us some of the advantages of the state tree.

Not a high priority, just something I've been thinking about since working with feedback and looking at how the different models interact.

Reply:
Could you open an RFC issue about it? It could be an agenda item to discuss for the week I'm in Oxford

Reply:
I've opened #603 for this.